.github/scripts/test-project.sh

```diff
@@ -37,7 +37,8 @@ if [ -f "prepare.sh" ]; then
   echo "prepare script found, executing $dir/$project/prepare.sh"
   echo ""
 
-  bash prepare.sh
+  # execute & allow export of env vars
+  . prepare.sh
```

Comment:
Uh, what does this do and how?

Reply:
Running with `bash` creates a sub-shell that inherits from the parent env. But doing `.` runs it in the same shell, allowing us to add env vars into the parent shell (because shell has env isolation per process).

Reply:
Do we use this for something already?

Reply:
Yep, for the `packagers/npm-global`